src/components/Modal.js

```diff
@@ -161,23 +149,9 @@ export default class Modal extends Component {
     ReactDOM.unmountComponentAtNode(this.node);
     const parent = getParentElement(this.props.parentSelector);
     parent.removeChild(this.node);
-
-    if (refCount.count() === 0) {
-      elementClass(document.body).remove(this.props.bodyOpenClassName);
-    }
   }
 
   renderPortal = props => {
-    if (props.isOpen || refCount.count() > 0) {
-      elementClass(document.body).add(this.props.bodyOpenClassName);
-    } else {
-      elementClass(document.body).remove(this.props.bodyOpenClassName);
-    }
-
-    if (props.ariaHideApp) {
-      ariaAppHider.toggle(props.isOpen, props.appElement);
-    }
-
     this.portal = renderSubtreeIntoContainer(this, (
```

Comment:
This is something that bugs me. Not sure what happens with `this.portal` when a modal is open and close too many times. I'll make some tests.
When Modal is unmounted, can this property leaks?